src/Data/Uniques/Special/New.lua

```diff
@@ -10,18 +10,93 @@ Faithguard
 Runic Helm
 League: Expedition
 Requires Level 25, 30 Str, 30 Dex, 30 Int
-+30 to Intelligence
```

Comment:
This file is intended for uniques that are revealed before league launch, where the modifier ranges are unknown. Once they are known, these uniques should be moved to the item type specific data files.